Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
```

Comment:
I'd like to point out here, in passing, that venting at the MacHomebrew Twitter account, calling our process "b/s" and referring to maintainers as "roaming neckbeards"  is not the way to speed up the review process. None of us are being paid for our work around Homebrew, we do the best we can around life's demands, and yes sometimes this means elements such as this are missed by one maintainer and picked up by another.

Before you contribute again please spend some time familiarising yourself with our [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODEOFCONDUCT.md).

<img width="552" alt="screen shot 2016-04-21 at 17 37 02" src="https://cloud.githubusercontent.com/assets/6998367/14717098/c24deef2-07e7-11e6-86aa-79e3dcf1ca92.png">